nav2_costmap_2d/src/costmap_2d_ros.cpp

```diff
@@ -71,6 +71,15 @@ Costmap2DROS::Costmap2DROS()
     "nav2_costmap_2d::InflationLayer"}
 {
   declare_parameter("map_topic", rclcpp::ParameterValue(std::string("map")));
+  /**
+   * @brief
```

Comment:
Remove. Again _only_ add the following to this function:

`is_lifecycle_follower_ = false;`

That is it. No comments, no parameters, nothing.